Package.md

```diff
@@ -6,49 +6,58 @@ Connects to Salesforce from Ballerina.
 
 ## Salesforce Data APIs
 
-Salesforce has a vast landscape of APIs since they follow an API-first approach to building features on the Salesforce Platform. This approach gives users flexibility to manipulate their data however they want. The most commonly used Salesforce Data APIs are REST API, SOAP API, Bulk API and Streaming API and together they make up the Salesforce Data API. 
+Salesforce has a vast landscape of APIs since they follow an API-first approach to building features on the Salesforce 
```

Comment:
Do we need a package.md in root directory? We already have one in /sfdc directory.